From ffb0bd8b378dfb83ffb0e282a308f441d9fe0bc1 Mon Sep 17 00:00:00 2001
From: Joel <joel@joel.tokyo>
Date: Sat, 30 Apr 2022 15:49:32 +1000
Subject: [PATCH] qtile: remove wlroots as it fails to build

I don't use qtile with wayland so I can remove wayland dependencies.
---
 pkgs/applications/window-managers/qtile/default.nix | 2 --
 1 file changed, 2 deletions(-)

diff --git a/pkgs/applications/window-managers/qtile/default.nix b/pkgs/applications/window-managers/qtile/default.nix
index d0bec27e215..f89d088e611 100644
--- a/pkgs/applications/window-managers/qtile/default.nix
+++ b/pkgs/applications/window-managers/qtile/default.nix
@@ -57,14 +57,12 @@ let
       pyxdg
       pygobject3
       pywayland
-      pywlroots
       xkbcommon
     ];
 
     buildInputs = [
       libinput
       wayland
-      wlroots
       libxkbcommon
     ];
 
-- 
2.31.1

